test_runner/fixtures/benchmark_fixture.py

```diff
@@ -120,6 +121,27 @@ def record_duration(self, metric_name):
 
         self.results.record(self.request.node.name, metric_name, end - start, 's')
 
+    def get_io_writes(self, pageserver) -> int:
+        """
+        Fetch the "cumulative # of bytes written" metric from the pageserver
+        """
+        # Fetch all the exposed prometheus metrics from page server
+        all_metrics = pageserver.http_client().get_metrics()
+        # Use a regular expression to extract the one we're interested in
+        writes = re.search(r"pageserver_disk_io_bytes{io_operation=\"write\"} (\d+)", all_metrics)
+        return int(writes.group(1))
+
+    @contextmanager
+    def record_pageserver_writes(self, pageserver, metric_name):
+        """
+        Record bytes written by the pageserver during a test.
+        """
+        before = self.get_io_writes(pageserver)
+        yield
+        after = self.get_io_writes(pageserver)
+
+        self.results.record(self.request.node.name, metric_name, round((after - before) / (1024 * 1024)), 'MB')
```

Comment:
> 1024

Just to be pedantic, 1 MB = 10^6 bytes and most probably what we want.

1 MiB = 1024 ^ 2 bytes and and is most probably not what we want, unless we're dealing with data storage 🙂

Reply:
Isn't it the other way around? I was under the impression that we usually need MiB, not MB.

Reply:
Yeah, we're on the same page for sure. People still use MB for MiB, though.

Reply:
The test currently reports disk usage in MB, meaning 1024*1024 bytes too. So this is consistent with that.

I agree we should change it to MiB, just to avoid the ambiguity, but let's do that as a separate PR.